docs/research/research_done_using_TARDIS/ads.ipynb

```diff
@@ -0,0 +1,918 @@
+       "1   https://ui.adsabs.harvard.edu/abs/2020ApJS..25...  \n",
+       "2   https://ui.adsabs.harvard.edu/abs/2020ApJ...89...  \n",
+       "3   https://ui.adsabs.harvard.edu/abs/2020MNRAS.49...  \n",
+       "4   https://ui.adsabs.harvard.edu/abs/2020ApJS..24...  \n",
+       "5   https://ui.adsabs.harvard.edu/abs/2020MNRAS.49...  \n",
+       "6   https://ui.adsabs.harvard.edu/abs/2020MNRAS.49...  \n",
+       "7   https://ui.adsabs.harvard.edu/abs/2020ApJ...89...  \n",
+       "8   https://ui.adsabs.harvard.edu/abs/2020ApJ...89...  \n",
+       "9   https://ui.adsabs.harvard.edu/abs/2020A&A...63...  \n",
+       "10  https://ui.adsabs.harvard.edu/abs/2020A&A...63...  \n",
+       "11  https://ui.adsabs.harvard.edu/abs/2019ApJ...88...  \n",
+       "12  https://ui.adsabs.harvard.edu/abs/2019Natur.57...  \n",
+       "13  https://ui.adsabs.harvard.edu/abs/2019MNRAS.48...  \n",
+       "14  https://ui.adsabs.harvard.edu/abs/2019LRCA.......  \n",
+       "15  https://ui.adsabs.harvard.edu/abs/2019ApJ...87...  \n",
+       "16  https://ui.adsabs.harvard.edu/abs/2019MNRAS.48...  \n",
+       "17  https://ui.adsabs.harvard.edu/abs/2019A&A...62...  \n",
+       "18  https://ui.adsabs.harvard.edu/abs/2019ApJ...87...  \n",
+       "19  https://ui.adsabs.harvard.edu/abs/2019Natur.56...  \n",
+       "20  https://ui.adsabs.harvard.edu/abs/2019A&A...62...  \n",
+       "21  https://ui.adsabs.harvard.edu/abs/2018A&A...62...  \n",
+       "22  https://ui.adsabs.harvard.edu/abs/2018MNRAS.48...  \n",
+       "23  https://ui.adsabs.harvard.edu/abs/2018ApJ...86...  \n",
+       "24  https://ui.adsabs.harvard.edu/abs/2018MNRAS.47...  \n",
+       "25  https://ui.adsabs.harvard.edu/abs/2018A&A...61...  \n",
+       "26  https://ui.adsabs.harvard.edu/abs/2018SSRv..21...  \n",
+       "27  https://ui.adsabs.harvard.edu/abs/2017MNRAS.47...  \n",
+       "28  https://ui.adsabs.harvard.edu/abs/2017Natur.55...  \n",
+       "29  https://ui.adsabs.harvard.edu/abs/2017ApJ...84...  \n",
+       "30  https://ui.adsabs.harvard.edu/abs/2017A&A...60...  \n",
+       "31  https://ui.adsabs.harvard.edu/abs/2017A&A...59...  \n",
+       "32  https://ui.adsabs.harvard.edu/abs/2016MNRAS.46...  \n",
+       "33  https://ui.adsabs.harvard.edu/abs/2016ApJ...83...  \n",
+       "34  https://ui.adsabs.harvard.edu/abs/2016MNRAS.46...  \n",
+       "35  https://ui.adsabs.harvard.edu/abs/2016A&A...58...  \n",
+       "36  https://ui.adsabs.harvard.edu/abs/2016JPhB...4...  \n",
+       "37  https://ui.adsabs.harvard.edu/abs/2016MNRAS.45...  \n",
+       "38  https://ui.adsabs.harvard.edu/abs/2016JPhB...4...  \n",
+       "39  https://ui.adsabs.harvard.edu/abs/2015MNRAS.45...  \n",
+       "40  https://ui.adsabs.harvard.edu/abs/2015MNRAS.45...  \n",
+       "41  https://ui.adsabs.harvard.edu/abs/2014MNRAS.44...  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df"
```

Comment:
I don't think we need multiple outputs from the notebook

Reply:
Would it be better to keep the string output that it displays inside of the notebook and remove the dataframe, or remove the string output that it displays inside of the notebook and keep the dataframe?